lib/wallaroo/core/metrics/reconnecting_metrics_sink.pony

```diff
@@ -75,6 +75,13 @@ actor ReconnectingMetricsSink
   var _service: String
   var _from: String
 
+  let _asio_flags: U32 =
+    ifdef not windows then
```

Comment:
Do you have any windows code elsewhere? If not, then I don't think the ifdef not windows is needed anywhere.

Reply:
@SeanTAllen We have ifdefs checking `not windows` elsewhere in the codebase where there is TCP-related code.  Whether those are necessary is a different question.

Reply:
If we remove the windows ifdef statements, then there's even more drift between our TCP-implementing code and the Pony runtime's version or the TCP-abstracting/simplifying work that John has started, e.g. PR #2906.  I'd prefer the ifdef removal to happen when #2906-like refactorings are made.

Reply:
given that we don't support windows nor do we have complete windows code available, i'd rather it was removed and we add it in if/when we aim to support windows and not until then.